pkg/generator/generator.go

```diff
@@ -381,3 +401,9 @@ func (g Generator) include(templateName string, templateData interface{}) (strin
 
 	return out.String(), nil
 }
+
+func (g Generator) logMessage(ctx context.Context, format string, params ...interface{}) {
+	if g.log != nil {
+		g.log.Debugf(ctx, format, params...)
+	}
```

Comment:
I have a couple of issues with this apprach:

1. When the log can be nil and why deviate from the usual logger name?
2. We already agreed in the SIG that we don't want to use logger in command line tools. I think this is the case. According to that the preference would be to simply fmt.Printf them.
3. This looks like very internal debugging information. I think it should not be printed by default. I'd say we should use some sort `--verbose` flag to enable those messages.